src/Swaps/ISwapper.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.0;
+
+/// @title ISwapper
+/// @custom:security-contact security@euler.xyz
+/// @author Euler Labs (https://www.eulerlabs.com/)
+/// @notice Interface of a helper contracts, which handle swapping of tokens for the EVK
+interface ISwapper {
+    /// @title SwapParams
+    /// @notice This struct holds all the parameters needed to carry out a swap
+    struct SwapParams {
+        // An id of the swap handler to use
+        bytes32 handler;
+        // Swap mode to execute
+        // 0 - exact input swap
+        // 1 - exect output swap
+        // 2 - exact output swap and repay, targeting a debt amount of an account
+        uint256 mode;
+        // An EVC compatible account address
+        address account;
+        // Sold asset
+        address tokenIn;
+        // Bought asset
+        address tokenOut;
+        // Vault to which the unused input in exact output swap will be deposited back
+        address vaultIn;
+        // In swapping modes (0 and 1) - address of the intended recipient of the bought tokens
+        // In swap and repay mode (2) - address of the liability vault of the account, where to repay debt
+        // Note that if the swap uses off-chain encoded payload, the receiver might be ignored. The swapper
+        // should verify the assets are in fact in the receiver address after the swap
+        address receiver; // in TARGET_DEBT liability vault
```

Comment:
in mode 0, I think that the `receiver` is ignored. this is because mode 0 only works with 1inch and UniswapAutoRouter which have the `receiver` encoded in `data`.

Reply:
The interface is supposed to be generic, so there might be a handler that needs a receiver in mode 0